target/linux/ramips/base-files/etc/board.d/01_leds

```diff
@@ -335,11 +335,19 @@ wndr3700v5)
 wt3020)
 	ucidef_set_led_default "power" "power" "$board:blue:power" "0"
 	;;
-y1|\
+y1)
+	ucidef_set_led_default "power" "power" "$board:blue:power" "1"
+	set_usb_led "$board:blue:usb"
+	set_wifi_led "$board:blue:wifi" "wlan1"
+	ucidef_set_led_netdev "wifi5g" "wifi5g" "$board:blue:wifi5g" "wlan0" "link tx rx"
+	ucidef_set_led_netdev "lan" "lan" "$board:blue:lan" "eth0.1" "tx rx"
+	ucidef_set_led_netdev "wan" "wan" "$board:blue:internet" "eth0.2" "tx rx"
```

Comment:
Gave it a quick test on a VGV7510KW22 and it works for me with the link parameter.

If I unplug the network cable:

```
[  213.263560] eth0: port 3 lost link
[  213.266285] br-lan: port 1(eth0.1) entered disabled state
```

And the configured LED is switched off.

There must be something else wrong on the Y1/Y1s.